src/compat/db_json.cpp

```diff
@@ -2681,7 +2754,7 @@ db_json_split_path_by_delimiters (const std::string &path, const std::string &de
     {
       if (path[end] == '"')
 	{
-	  std::size_t index_of_closing_quote = path.find_first_of ("\"", end + 1);
+	  std::size_t index_of_closing_quote = path.find_first_of ('"', end + 1);
```

Comment:
Just double checking - you're sure `"` does not require escaping even as character token?

Reply:
Yes, as character there is no need for escaping